ppr-ui/src/components/common/QsAccessBtn.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div id="qs-access-btn">
+    <template v-if="isRoleQualifiedSupplier">
+      <a class="approved-qs-link">
+        <v-icon left color="primary" class="fs-21 mt-n1">mdi-account-lock</v-icon>
+        <span class="primary--text fs-13">Approved Qualified Supplier</span>
+      </a>
+    </template>
+    <template v-else>
+      <v-tooltip
+        top
+        class="request-qs-tooltip"
+        content-class="top-tooltip pa-4 mr-2"
+        transition="fade-transition"
+        :disabled="!isPendingQsAccess"
+      >
+        <template v-slot:activator="{ on }">
+          <a
+            v-on="on"
+            class="request-qs-link"
+            :class="{'pending-qs-link': isPendingQsAccess || isUserAccessRoute }"
+            @click="goToUserAccess()"
+          >
+            <v-icon left color="primary" class="fs-21 mt-n1">mdi-account-lock</v-icon>
+            <span class="primary--text fs-13">Request MHR Qualified Supplier Access</span>
+          </a>
+        </template>
+        This account has a Qualified Supplier application that is under review.
+      </v-tooltip>
+    </template>
+  </div>
+
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue-demi'
+import { storeToRefs } from 'pinia'
+import { useStore } from '@/store/store'
+import { useUserAccess } from '@/composables'
+
+export default defineComponent({
+  name: 'QsAccessBtn',
```

Comment:
Would be helpful to have longer name, but I see that it's used a lot throughout, so we can keep it.